x-pack/plugins/lens/public/metric_visualization_plugin/metric_expression.tsx

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { ExpressionFunction } from 'src/legacy/core_plugins/interpreter/types';
+import { EuiFlexGroup, EuiFlexItem, EuiText } from '@elastic/eui';
+import { MetricConfig } from './types';
+import { KibanaDatatable } from '../types';
+import { RenderFunction } from './plugin';
+
+export interface MetricChartProps {
+  data: KibanaDatatable;
+  args: MetricConfig;
+}
+
+export interface MetricRender {
+  type: 'render';
+  as: 'lens_metric_chart_renderer';
+  value: MetricChartProps;
+}
+
+export const metricChart: ExpressionFunction<
+  'lens_metric_chart',
+  KibanaDatatable,
+  MetricConfig,
+  MetricRender
+> = ({
+  name: 'lens_metric_chart',
+  type: 'render',
+  help: 'A metric chart',
+  args: {
+    title: {
+      types: ['string'],
+      help: 'The chart title.',
```

Comment:
This help text exists to help users who are manually writing out these expressions, right? It should probably be i18n

Reply:
Good point. It's currently localized in half of Kibana and not in others. I'll update this PR to include localization of all our expression functions.